wrappers/java/src/test/java/org/hyperledger/indy/sdk/utils/InitHelper.java

```diff
@@ -3,12 +3,10 @@
 
 import org.hyperledger.indy.sdk.LibIndy;
 
-import java.io.File;
-
 public class InitHelper {
 	public static void init() {
 
-		if (!LibIndy.isInitialized()) LibIndy.init(new File("./lib/libindy"));
+		if (!LibIndy.isInitialized()) LibIndy.init("./lib/");
```

Comment:
I am not sure that looking for libindy inside jar is a good idea at all. libindy should be installed with corresponded installation package and loaded from system LD path.